Packs/CrowdStrikeFalcon/Playbooks/playbook-Command_and_Scripting_Interpreter.yml

```diff
@@ -0,0 +1,1717 @@
+              complex:
+                root: CommandLineAnalysis
+                filters:
+                - - operator: in
+                    left:
+                      value:
+                        simple: CommandLineAnalysis.original_command
+                      iscontext: true
+                    right:
+                      value:
+                        simple: incident.cmdline
+                      iscontext: true
+                accessor: score
+            iscontext: true
+          right:
+            value:
+              simple: "26"
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1220,
+          "y": 1850
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "77":
+    id: "77"
+    taskid: 2bf225f1-f4b7-49fd-8309-e857501d789c
+    type: regular
+    task:
+      id: 2bf225f1-f4b7-49fd-8309-e857501d789c
+      version: -1
+      name: Terminate Process
+      description: Execute an active responder kill command on a single host.
+      script: '|||cs-falcon-rtr-kill-process'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#error#':
+      - "78"
+      '#none#':
+      - "17"
+    scriptarguments:
+      host_id:
+        complex:
+          root: incident
+          accessor: agentsid
+          transformers:
+          - operator: uniq
+          - operator: FirstArrayElement
+      process_ids:
+        complex:
+          root: incident
+          accessor: pid
+          transformers:
+          - operator: uniq
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: errorPath
+    view: |-
+      {
+        "position": {
+          "x": 1470,
+          "y": 2020
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "78":
+    id: "78"
+    taskid: 40dd73b5-d156-4b22-84bd-b5b09f0cb144
+    type: condition
+    task:
+      id: 40dd73b5-d156-4b22-84bd-b5b09f0cb144
+      version: -1
+      name: Manual Response
+      description: Manual Response - Analyst actions is required.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "17"
+      Isolate Endpoint:
+      - "81"
+      Terminate Parent Process:
+      - "104"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 970,
+          "y": 2250
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    message:
+      to:
+      subject:
+      body:
+        simple: Dear analyst, we've identified suspicious behavior and the playbook could not handle this alert automatically. Therefore, please choose the right remediation action.
```

Comment:
this is unclear.
If we cant handle this alert automatically, how come you are running the automations after the analyst choose the action?

Reply:
Changed "handle" to remediate". How does that sound to you?